tutorial/README.md

```diff
@@ -0,0 +1,476 @@
+
+When changes are done on a scenario, before doing lengthy calculations on it, it is important to test that it works as expected and give logical results. This can be done by testing some trip calculations on the transit network.
+
+Navigate to the `Routing` panel, by clicking on the `Routing` in the left menu.
+
+![Lesson 3: routing menu](images/lesson3_routingMenu.png)
+
+You can fine-tune the parameters, to change the departure/arrival time, maximum travel time, minimum waiting time, maximum access, egress and transfer time.
+
+Then select the scenario to test and whether you want alternatives or not.
+
+To define the origin and destination, you may simply click on the map. The first click will define the origin, all further clicks will change the destination.
+
+To change the origin again, right-click on the map. A popup menu will open and allow to select the origin or destination for the current point localtion.
+
+Finally, click on the ![calculation button](images/buttonSave.png) calculation button and you should see the result of the route display on the map.
+
+![Lesson 3: display routing results](images/lesson3_displayRoutingResults.png)
+
+When selecting results with alternatives, you can navigate through the alternatives to see the detailed results for each.
+
+![Lesson 3: routing results with alternatives](images/lesson3_resultsWithAlternatives.png)
+
+### Troubleshooting routing
+
+The routing engine returns no result? 
+
+If there is no access, either at origin or destination (no stop nodes near those points), or if there is no service at origin or destination, the error message will be explicit enough. You can change the origin or destination for locations where there is service available.
+
+![Lesson 3: routing results error no access](images/lesson3_resultsErrorNoAccess.png)
+![Lesson 3: routing results error no service](images/lesson3_resultsErrorNoService.png)
+
+If the message says it returned no result, it will hint to look at some parameters values. You can try changing the travel time, access/egress, etc.
+
+![Lesson 3: routing results error no routing](images/lesson3_resultsErrorNoRouting.png)
+
+If you are certain that there should be a route with the given parameters, go back to [lesson 2](#lesson-2-prepare-imported-data-for-calculations) and validate that the scenario contains the expected lines, then refresh calculation data to make sure it is up to date.
+
+## Lesson 4: Visualize accessibility maps
+
+Another type of calculation that can be done with the scenario is the accessibility maps. Accessility maps are isochrones displaying the region accessible in transit within a certain amount of time.
+
+Navigate to the `Accessibility map` panel, by clicking on the `Accessibility map` in the left menu.
+
+![Lesson 4: accessibility map menu](images/lesson4_accessibilityMapMenu.png)
+
+Many of the parameters to set are similar to those of the `Routing`: departure or arrival time and other trip times.
```

Comment:
We should add that setting a departure time will make the point the origin and the arrival time the destination. (Somewhat explicit, but it's good to be clear)